kubernetes/database.libsonnet

```
@@ -37,6 +37,8 @@ local k = import 'kubernetes/kube.libsonnet';
     local this = self,
     // bento is the bento which contains the database cluster and application
     bento:: error 'bento is required',
+    // database_name is the name of the application database to create within the cluster
```

I think it's helpful to try to describe the namespace win this comment, e.g.:

```suggestion
    // database_name is the name of the database to create within the postgresql cluster. Schemas for the application will be created in the provided datatabase. See https://www.postgresql.org/docs/current/ddl-schemas.html
```